pandas/io/html.pyi

```diff
@@ -25,7 +25,7 @@ def read_html(
     index_col: Optional[Union[int, Sequence[Any]]] = ...,
     skiprows: Optional[Union[int, Sequence[Any], slice]] = ...,
     attrs: Optional[Mapping[str, str]] = ...,
-    parse_dates: bool = ...,
+    parse_dates: Union[bool, Sequence[Union[int, str, Sequence[Union[int, str]]]], Dict[str, Sequence[Union[int, str]]]] = ...,
```

Comment:
Does this strictly need to be a `dict`, or can it be a `Mapping` like `attrs`?

Reply:
To answer your question, I have a check on pandas' source code. And it seems the answer is yes. [This line](https://github.com/pandas-dev/pandas/blob/cc5eba9f5f9d833f3df7afff5f20b51c0cd3ee3d/pandas/io/parsers/base_parser.py#L1227) dictates that we can't use `Mapping` to replace `Dict`.

Reply:
Thanks for checking!